File: src/main/java/com/akagiyui/drive/model/request/CreateFolderRequest.java
package com.akagiyui.drive.model.request;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

/**
 * 创建文件夹 请求实体
 *
 * @author AkagiYui
 */
@Data
public class CreateFolderRequest {
    /**
     * 文件夹名
     */
    @NotBlank
    private String name;

    /**
     * 父文件夹ID
     */
    private String parent;
}

File: app/src/main/kotlin/com/akagiyui/drive/model/request/CreateFolderRequest.kt
package com.akagiyui.drive.model.request

/**
 * 创建文件夹 请求实体
 *
 * @author AkagiYui
 */
class CreateFolderRequest {
    /**
     * 文件夹名
     */
    lateinit var name: String

    /**
     * 父文件夹ID
     */
    var parent: String? = null
}
